src/EditorFeatures/Core/Implementation/EncapsulateField/AbstractEncapsulateFieldCommandHandler.cs

```diff
@@ -102,7 +92,7 @@ private bool Execute(EncapsulateFieldCommandArgs args, IWaitContext waitContext)
                     return false;
                 }
 
-                waitContext.AllowCancel = false;
+                waitScope.AllowCancellation = false;
 
                 var finalSolution = result.GetSolutionAsync(cancellationToken).WaitAndGetResult(cancellationToken);
```

Comment:
>cancellationToken [](start = 60, length = 17)

here is another case, the cancellation is disabled but still using token.
Also I don't really understand why we disable cancellability here, but not in AbstractChangeSignatureCommandHandler for example.

Reply:
but I'm going to leave it as is unless anybody objects

---
In reply to: [160064271](https://github.com/dotnet/roslyn/pull/23769#discussion_r160064271) [](ancestors = 160064271)

Reply:
I filed https://github.com/dotnet/roslyn/issues/24153 to track it.